doc/api/process.md

```diff
@@ -1499,8 +1499,11 @@ debugger. See [Signal Events][].
 ## `process.mainModule`
 <!-- YAML
 added: v0.1.17
+deprecated: REPLACEME
 -->
 
+> Stability: 0 - Deprecated: Use [`require.main`][] instead.
+
```

Comment:
It would be worth briefly mentioning the reason why either here or in the deprecation doc.

Side point: we should start linking to the deprecation doc from these stability statements, and start including the deprecation code.